tests/selectors/wallet.selectors.ts

```diff
@@ -0,0 +1,5 @@
+export enum WalletSelectors {
```

Comment:
Let's avoid the term wallet, as it's very vague. These seem like home page selectors?

Reply:
No, these really aren't just home page related ...what is a higher level word to use?